omniscidb/QueryEngine/RelAlgExecutor.h

```diff
@@ -236,7 +242,9 @@ class RelAlgExecutor {
   Executor* executor_;
   std::unique_ptr<hdk::ir::QueryDag> query_dag_;
   std::shared_ptr<SchemaProvider> schema_provider_;
+  Data_Namespace::DataMgr* data_mgr_;
```

Comment:
Well, I don't have to actually hold it here, it is used only to get the registry. We can add the registry as a new ctor argument to avoid using `DataMgr`, but for now I don't want to break PyHDK API forcing the registry initialization by users. The plan is to add registry initialization in Modin and other places and then remove the registry creation from the `RelAlgExecutor` ctor.